apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -466,6 +466,10 @@ Optional arguments:
                         indirectly depend on the specified project
   -v, --verbose         Display the logs during the build, rather than just 
                         displaying the build status summary
+  -o, --changed-projects-only
```

Comment:
This probably shouldn't be an option for `rebuild`.

Reply:
agreed, but I think that's out of scope here? I think this file was auto-generated when I went to run tests and might just be outdated on master.